### PR TITLE
[io-cassandra] Support for PlainTextAuthProvider

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraDBAccessor.java [hunk 1]
```diff
@@ -229,12 +251,21 @@ private Cluster getCluster(String authority) {
...
+
+  @VisibleForTesting
+  Builder configureClusterBuilder(Builder builder, String authority) {
+
+    builder.addContactPointsWithPorts(
+        Arrays.stream(authority.split(","))
+            .map(CassandraDBAccessor::getAddress)
+            .collect(Collectors.toList()));
+    if (username != null) {
+      Preconditions.checkArgument(password != null, "Password must be specified.");
```
Comment (je-ik) [inline direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraDBAccessor.java:265]: Is it possible that password can be empty? Can we provide empty password in the configuration, or will it be `null` here?

**Review by je-ik — APPROVED** [review]
nit: not sure how this plays with empty passwords.

File: direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraDBAccessor.java (hunk 1 above)
Comment (LesTR) [reply direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraDBAccessor.java]: Make sense, changed.

**Review by je-ik — APPROVED** [review]
👍

**Pull request merged**